web/profiles/custom/os2loop/modules/os2loop_search_db/os2loop_search_db.services.yml

```diff
@@ -1,3 +1,5 @@
 services:
-  os2loop_search_db.search_api_autocomplete_helper:
-    class: Drupal\os2loop_search_db\Helper\SearchApiAutocompleteHelper
+  Drupal\os2loop_search_db\Helper\Helper:
```

Comment:
Why "Drupal\"

Reply:
It's the FQN of the helper class and makes it possible to write `Drupal::service(Helper::class)` in the `.module` file.